airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_errors.py

```diff
@@ -300,6 +303,17 @@ class TestRealErrors:
                 # Potentially could be caused by some particular field (list of requested fields is constant).
                 # But since sync was successful on next attempt, then conclusion is that this is a temporal problem.
             ),
+            (
+            "error_503_service_unavailable",
+            {
+                "json": {
+                    "error": {
+                        "message": "Call was not successful",
+                    }
+                },
+                "status_code": 503,
+            },
+        ),
         ],
```

Comment:
this indentation is looking weird!
```suggestion
            ),
        ],
```

Reply:
My bad, thanks! Fixed 👍